pages/activeStudents.vue

```diff
@@ -75,13 +86,32 @@ export default {
   computed: {
     searchStudentResult() {
```

Comment:
Ao que me parece a pesquisa é local da página. Se um estudante for de outra página da API a pesquisa dá errado.

Reply:
Fica para a Meta 5 (#26)